Firestore/Example/Tests/Integration/API/FIRValidationTests.mm

```diff
@@ -704,15 +704,15 @@ - (void)expectFieldPath:(NSString *)fieldPath toFailWithReason:(NSString *)reaso
 
 - (void)verifyExceptionForInvalidLatitude:(double)latitude {
   NSString *reason = [NSString
-      stringWithFormat:@"GeoPoint requires a latitude value in the range of [-90, 90], but was %f",
+      stringWithFormat:@"GeoPoint requires a latitude value in the range of [-90, 90], but was %g",
```

Comment:
The test was expecting something like "180.100000" but the new error was coming out as "180.1" (which is better!)